actionview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   `ActionView::Base.annotate_template_file_names` only annotates HTML output.
```

Comment:
I don't think this changelog is necessary since the previous entry doesn't say it works for everything.